<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>button</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="EnhancedChartViewSystem" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>441</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>显示极大值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>左移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>添加注释</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>放缩图表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>显示极小值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>显示峰值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>右移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空点值框</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>CALLOUT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>OPENGL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>添加带光标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>添加线光兵</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnhancedChartViewSystem</class>
   <extends>QGraphicsView</extends>
   <header>enhancedchartviewsystem.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
